index.js

```diff
@@ -349,10 +349,12 @@ class Raspi extends EventEmitter {
 
   digitalWrite(pin, value) {
     const pinInstance = this[getPinInstance](this.normalize(pin));
-    if (pinInstance.mode != OUTPUT_MODE) {
+    if (pinInstance.mode === INPUT_MODE && value === HIGH) {
+      pinInstance.peripheral = new DigitalInput({ pin: this.normalize(pin), pullResistor: PULL_DOWN });
```

Comment:
Oh one other thing, do you think we could add a flag on the pinInstance object that indicates if we've already enabled the pull down resistor? That way, we don't have to create new objects all the time.